valkey.conf

```diff
@@ -174,6 +174,16 @@ timeout 0
 # On other kernels the period depends on the kernel configuration.
 tcp-keepalive 300
 
+# TCP keepalive probes.
+#
+# The number of TCP keepalive probes to send before considering the connection dead.
+# Only valid if tcp-keepalive is non-zero.
+# This is useful below situations:
+#
+# 1) Faster detection of dead peers.
+# 2) Avoiding premature disconnection due to network equipment in the middle.
+tcp-keepalive-probes 3
+
```

Comment:
IMO tt's a bit verbose. You can simplify it while keeping the essential details:
`# Number of TCP keepalive probes before closing a dead connection. `

Reply:
[e24f3f9](https://github.com/valkey-io/valkey/pull/1765/commits/e24f3f9715c22ef960a26f14e2fa22d86e6016b3)